### ISSUE TITLE
Claim blocks are not given to players

### ISSUE THREAD

**Issue by ilipily** [GriefPrevention/GriefPrevention#399]
### What happened:
While being online, you'll get 120 claim blocks/hour. Now all the player won't get any claim blocks.

### What you expected:
That all the player get their 120 claim blocks/hour.

### Steps to reproduce:
I don't know why it happened or what it caused but a few days ago it worked normal.

### Server and GriefPrevention version:
___This can be found by running `/version` and `/version GriefPrevention` on your server.___
Paste of `/version`:  git-Spigot-dcd1643-d7bebef (MC: 1.12.2)
Paste of `/version GriefPrevention`: 16.8

### Stack trace/error or server log
There are no errors in the config.
```
https://gist.github.com/ilipily/6c3cfb7acf78072776942bd4f70938ed
```

### GriefPrevention config.yml

```yml

# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/
GriefPrevention:
  SeaLevelOverrides:
    survival: -1
    survival_nether: -1
    survival_the_end: -1
  Claims:
    Mode:
      survival_nether: Disabled
      survival_the_end: Disabled
      survival: Survival
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: true
    LockTrapDoors: true
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    ProtectHorses: true
    InitialBlocks: 100
    Claim Blocks Accrued Per Hour:
      Default: 120
    Max Accrued Claim Blocks:
      Default: -1
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 4
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: 0
    InvestigationTool: STICK
    ModificationTool: GOLD_SPADE
    Expiration:
      ChestClaimDays: 7
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: -1
        ExceptWhenOwnerHasBonusClaimBlocks: -1
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    PortalGenerationRequiresPermission: false
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
  Spam:
    Enabled: true
    LoginCooldownSeconds: 60
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: false
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 10
  PvP:
    RulesEnabledInWorld:
      survival: true
      survival_nether: true
      survival_the_end: true
    ProtectFreshSpawns: true
    PunishLogout: true
    CombatTimeoutSeconds: 15
    AllowCombatItemDrop: false
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: true
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers: true
    AllowFlintAndSteelNearOtherPlayers: true
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: true
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: true
  LimitSkyTrees: true
  LimitTreeGrowth: false
  LimitPistonsToLandClaims: true
  FireSpreads: false
  FireDestroys: false
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  SmartBan: true
  Mute New Players Using Banned Words: true
  MaxPlayersPerIpAddress: 3
  Siege:
    Worlds: []
    BreakableBlocks:
    - DIRT
    - GRASS
    - LONG_GRASS
    - COBBLESTONE
    - GRAVEL
    - SAND
    - GLASS
    - THIN_GLASS
    - WOOD
    - WOOL
    - SNOW
    DoorsOpenDelayInSeconds: 300
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: false
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Mods:
    BlockIdsRequiringAccessTrust: []
    BlockIdsRequiringContainerTrust:
    - 99999:*:Example - ID 99999, all data values.
    BlockIdsExplodable: []
    PlayersIgnoringAllClaims: []
  Advanced:
    fixNegativeClaimblockAmounts: true
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false


```

### Plugin list (if applicable):
https://gyazo.com/2a810ab3e38f78131a1f21dcf351210a

**Comment by RoboMWM** [comment]
Enable debug logging, and check gp logs after a while of player activity for block accrual messages.

**Comment by ilipily** [comment]
I found the error causing the players getting no claim blocks. I used the **-1** to let the plugin know they can get an unlimited amount of claim blocks, but it says on that line that the player won't get more claim blocks if they have over the **-1** blocks.
```yml

    Max Accrued Claim Blocks:
      Default: -1

```
So this can't be **-1**. Is there any other way I can state that the player can get as many blocks as he/she is online?
